dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -226,30 +237,25 @@ static Configuration configureFreemarker() {
         for(String coordinates: jarToBomMembers.get(jar)) {
           Set<ClassFile> classFilesForProblem = symbolProblemTable.get(coordinates, problem);
           if (classFilesForProblem == null) {
-            classFilesForProblem = new HashSet<>();
+            classFilesForProblem = Sets.newHashSet();
```

Comment:
Per API docs "Note for Java 7 and later: this method is now unnecessary and should be treated as deprecated. Instead, use the HashSet constructor directly, taking advantage of the new "diamond" syntax."

Reply:
Now it uses `ImmutableSet.Builder` to have deterministic iteration order.